std/array_list.zig

```diff
@@ -39,6 +39,12 @@ pub fn ArrayList(comptime T: type) -> type{
             *new_item_ptr = *item;
         }
 
+        pub fn appendSlice(l: &Self, items: []const T) -> %void {
+            for (items) |item| {
```

Comment:
This seems like a good case to call `ensureCapacity` once, then do an unchecked memcpy.